### PR TITLE
Grade school

### PR DESCRIPTION
**Pull request by kmarker1101**

### CODE REVIEW

**Comment by BNAndras** [comment]
This one is a little fiddly with all the reimplemented tests to look out for but I’ll give you a review by the end of the weekend.

File: exercises/practice/grade-school/grade-school-test.el
Comment (BNAndras) [inline exercises/practice/grade-school/grade-school-test.el:1]: `(should (equal ... t)`  should be `(should ...)`. Conversely `(should (equal ... nil)` should be `(should-not ...)`.

**Review by BNAndras — CHANGES_REQUESTED** [review]

File: exercises/practice/grade-school/grade-school-test.el
Comment (kmarker1101) [reply exercises/practice/grade-school/grade-school-test.el:1]: will fix. Thanks

Comment (kmarker1101) [reply exercises/practice/grade-school/grade-school-test.el:1]: Think I got them all. Thanks

**Review by BNAndras — APPROVED** [review]

**Pull request merged**